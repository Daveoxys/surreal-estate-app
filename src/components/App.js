import React from "react";
import "../styles/App.css";

const App = () => {
  return (
    <div className="App">
      <h2>Surreal Estate</h2>
    </div>
  );
};

export default App;
