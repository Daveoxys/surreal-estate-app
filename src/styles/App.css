@import url("https://fonts.googleapis.com/css2?family=Baloo+Bhai+2&display=swap");
body {
  font-family: "Baloo Bhai 2", cursive;
  font-size: 20px;
  background-color: rgba(43, 0, 28, 0.226);
  margin: 0%;
  padding: 0%;
}
