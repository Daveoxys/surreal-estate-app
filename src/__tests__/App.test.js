import React, { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom/cjs/react-router-dom.min";
import App from "../components/App";

describe("App", () => {
  const { asFragment } = render(
    <BrowserRouter>
      <App />
    </BrowserRouter>
  );

  it("renders correctly", () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
