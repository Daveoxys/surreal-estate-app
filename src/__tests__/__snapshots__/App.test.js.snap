// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="navbar"
    >
      <ul
        class="navbar-links"
      >
        <img
          alt="surreal-logo"
          class="navbar-logo"
          data-testid="logo"
          src="logo.png"
        />
        <a
          class="navbar-links-item"
          href="/"
        >
          View Properties
        </a>
        <a
          class="navbar-links-item"
          href="/add-property"
        >
          Add a Property
        </a>
      </ul>
    </div>
    <div
      class="properties"
    >
      <p>
        Properties Page
      </p>
    </div>
  </div>
</DocumentFragment>
`;
